.github/workflows/publish.yaml

```diff
@@ -9,5 +9,6 @@ jobs:
     uses: bakdata/ci-templates/.github/workflows/python-poetry-publish-pypi.yaml@1.40.4
     with:
       publish-to-test: false
+      poetry-version: "1.4.1"
```

Comment:
I just saw the timeout env var was only added in Poetry 1.5.0. https://github.com/python-poetry/poetry/pull/7081
Does it make sense to update to the latest version?

Reply:
It make sense to have that above 1.5.0. But we still have the issue with [poetry and windows](https://github.com/bakdata/kpops/pull/430/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddL31). I can give it a try with the newest version (1.7.1)

Reply:
It seems that the problem with the windows runners is not occurring. We can keep the 1.7.1